### PR TITLE
Simplify dob backup workflow

### PR DESCRIPTION
**Pull request by shikharish**

### CODE REVIEW

**Comment by shikharish** [comment]
cc: @rajivharlalka @proffapt

File: backup/run_backup.sh [hunk 1]
```diff
@@ -50,4 +50,4 @@ if ! $backup_to_dropbox $backup_file; then
 	exit 1
 fi
 
-echo -e "\n-- Database of Babel Backup and Rotation END --\n"
\ No newline at end of file
+echo -e "\n-- Backup and Rotation finisned at `date +"%Y/%m/%d %H:%M:%S"` --\n"
```
Comment (rajivharlalka) [inline backup/run_backup.sh]:
```suggestion
echo -e "\n-- Backup and Rotation finished at `date +"%Y/%m/%d %H:%M:%S"` --\n"
```

**Review by rajivharlalka — DISMISSED** [review]

**Comment by proffapt** [comment]
I will review it tomorrow, ping me once tomorrow. Thanks.

File: Dockerfile [hunk 2]
```diff
@@ -1 +1,9 @@
 FROM postgres:16
+
+RUN apt-get update && apt-get install -y curl python3 python3-pip gzip
+
+WORKDIR /root
+
+COPY ./backup ./backup
+
+RUN pip3 install -qr ./backup/requirements.txt --break-system-packages
```
Comment (proffapt) [inline Dockerfile]:
```suggestion
WORKDIR /backup

COPY ./backup .

RUN pip3 install -qr requirements.txt --break-system-packages
```

File: docker-compose.yml [hunk 3]
```diff
@@ -13,32 +13,23 @@ services:
...
       - DROPBOX_ACCESS_TOKEN=${DROPBOX_ACCESS_TOKEN}
       - DROPBOX_REFRESH_TOKEN=${DROPBOX_REFRESH_TOKEN}
       - SLACK_INCIDENTS_WH_URL=${SLACK_INCIDENTS_WH_URL}
     volumes:
       - db-volume:/var/lib/postgresql/data
+    healthcheck:
+      test:
+        [
+          "CMD-SHELL",
+          "/root/backup/run_backup.sh >> /var/log/backups.log 2>&1 || exit 1",
```
Comment (proffapt) [inline docker-compose.yml]:
```suggestion
          "/root/backup/run_backup.sh >> /var/log/pg_dump.log 2>&1 || exit 1",
```

File: docker-compose.yml [hunk 4]
```diff
@@ -13,32 +13,23 @@ services:
...
+        ]
+      interval: 24h
+      timeout: 30s
```
Comment (proffapt) [inline docker-compose.yml]:
```suggestion
      timeout: 120s
```

**Review by proffapt — CHANGES_REQUESTED** [review]
- [x] remove crontab file

**Review by proffapt — APPROVED** [review]

**Pull request merged**